bin/next-start

```diff
@@ -10,11 +10,13 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'production'
 const argv = parseArgs(process.argv.slice(2), {
   alias: {
     h: 'help',
+    H: 'hostname',
     p: 'port'
   },
   boolean: ['h'],
   default: {
-    p: 3000
+    p: 3000,
+    hostname: 'localhost'
```

Comment:
> If the hostname is omitted, the server will accept connections on any IPv6 address (::) when IPv6 is available, or any IPv4 address (0.0.0.0) otherwise.

@nkzawa is right 😄